### ISSUE TITLE
Question: Undo "close window except current"

### ISSUE THREAD

**Issue by mattbdc** [helix-editor/helix#4772]
Imagine a scenario where you have many splits (a large left, and maybe a right split vertically with the bottom right into 2 horizontal - i.e. not a simple equal width 2 panes), and you want to focus on one - the old ctrl+w o works fine, but how do you revert this? vscode is dead ended here as they have no getLayoutEditor function in their clumsy API. i.e. how do you zen-in on one pane, do some work and go back to complicated layout using the helix commands? 

tl;dr How would I accomplish Undo "close window except current"  with helix?

**Comment by archseer** [comment]
This looks like a duplicate of https://github.com/helix-editor/helix/issues/2262
